src/components/variant-select.js

```diff
@@ -6,7 +6,19 @@ import {LinkNoUnderline} from './link'
 import useLocationChange from '../hooks/use-location-change'
 
 const VariantItem = ({title, shortName, url, active}) => (
-  <ActionList.Item as={LinkNoUnderline} to={url} state={{scrollUpdate: false}} id={shortName} active={active}>
+  <ActionList.Item
+    sx={{
+      ':hover:not([aria-disabled])': {
+        backgroundColor: '#666D75',
+        color: '#FFFFFF',
+      },
+    }}
```

Comment:
Since this site uses `@primer/react` now, we should try to make all color changes using predefined colors in theme or by setting colors in `src/theme.js`. See the `@primer/react` theming docs for examples on how to do this: https://primer.style/react/theming